style.css

```diff
@@ -163,3 +163,32 @@ body {
     display: inline;
   }
 }
+
+@keyframes fade-slide-down {
+  0% {
+    opacity: 0;
+    transform: translateY(-4rem);
+  }
+  100% {
+    opacity: 1;
+    transform: none;
+  }
+}
+
+@keyframes fade-slide-up {
+  0% {
+    opacity: 0;
+    transform: translateY(4rem);
+  }
+  100% {
+    opacity: 1;
+    transform: none;
+  }
+}
+.fade-in {
+  animation: fade-slide-down 2s ease-out forwards;
```

Comment:
```suggestion
  animation: fade-slide-down 0.6s ease-out forwards;
```

Reply:
also add a CSS block where for buttons (options) the animation has 0.2s delay. This can be achieved with animation-delay property.

Reply:
hi i have updated all other changes could not understand this one should i add a new class for button animations which have a delay of 0.2s.